File: 1421-find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function(nums) {
    let ans = 0;
    for(let num of nums){
        if((num >= 10 && num <= 99) || (num >= 1000 && num <= 9999) || num == 100000){
            ans++;
        }
    }
    return ans;
};

File: 1421-find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int n = nums.length;
        int ans = 0;
        for(int num : nums){
            int count = 0;
            while(num != 0){
                num /= 10;
                count++;
            }
            if(count % 2 == 0){
                ans++;
            }
        }
        return ans;
    }
}